asset/cloud-client/quickstart_createfeed_test.py

```diff
@@ -30,6 +32,7 @@
 TOPIC = 'topic-{}'.format(int(time.time()))
 
 
+@pytest.mark.skip(reason='TODO(cwxie-google): either remove or fix')
```

Comment:
Is this broken (as well as the others) as a result of the updates?

Reply:
Good question, I think I disabled the test because it was broken. I'll double-check.